### PR TITLE
{Monitor} `az monitor metrics alert update`: fix update action issue

### PR DESCRIPTION
**Pull request by kairu-ms**
Closed https://github.com/Azure/azure-cli/issues/17268

`webhook_properties` property has been renamed in SDK. In order to avoid this kind of breaking change, I rewrite the code to eliminate the dependency on this property.

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Monitor

File: src/azure-cli/azure/cli/command_modules/monitor/operations/metric_alert.py [hunk 1]
```diff
@@ -88,14 +88,9 @@ def update_metric_alert(instance, scopes=None, description=None, enabled=None, t
...
-            match = next(
-                (x for x in instance.actions if action.action_group_id.lower() == x.action_group_id.lower()), None
-            )
-            if match:
-                match.webhook_properties = action.webhook_properties
-            else:
-                instance.actions.append(action)
+        add_action_ids = {x.action_group_id.lower() for x in add_actions}
+        instance.actions = [x for x in instance.actions if x.action_group_id.lower() not in add_action_ids]
+        instance.actions.extend(add_actions)
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/operations/metric_alert.py:93]: No need to use `match` for `webhook_properties` now? What does this change mean?

Comment (kairu-ms) [reply src/azure-cli/azure/cli/command_modules/monitor/operations/metric_alert.py:93]: For 'add_actions', the logical of original code is that if that action already exist then update its properties. Now I change the logic, if that action already exist, remove the old one and add the new one.

**Review by msyyc — APPROVED** [review]

**Pull request merged**